src/components/board/Board.js

```diff
@@ -22,9 +22,11 @@ import WildlingCard from '~/components/wildling-card/WildlingCard';
 import { fetchGame, updateAll } from '~/redux/actions/';
 
 @connect(
-  (state, props) => ({
-    game: build(state, 'games', props.gameId) || {},
-  }),
+  (state, props) => {
+    return {
+      game: build(state, 'game', props.gameId, { eager: true }) || {},
+    };
+  },
```

Comment:
You are returning here just because you had to use the `()` so you could break the link on the Object, right? The previous version should work, no?

Reply:
This sintaxe is only for debug purpose,  the previous one work too if change games to singular